docs/config-description.md

```diff
@@ -1,452 +1,600 @@
-### Key: forwarding.request_timeout
+- **Default value**: `0`
+- **Constraints**: The value must be a positive double number.
+- **Description**: Specifies the timeout duration (in seconds) for the forwarding cache used in `rippled` communication.
+
+### forwarding.request_timeout
+
 - **Required**: True
 - **Type**: double
-- **Default value**: 10
-- **Constraints**: The value must be a positive double number
- -   **Description**: Timeout duration for the forwarding request used in Rippled communication.
-### Key: rpc.cache_timeout
+- **Default value**: `10`
+- **Constraints**: The value must be a positive double number.
+- **Description**: Specifies the timeout duration (in seconds) for the forwarding request used in `rippled` communication.
+
+### rpc.cache_timeout
+
 - **Required**: True
 - **Type**: double
-- **Default value**: 0
-- **Constraints**: The value must be a positive double number
- -   **Description**: Timeout duration for RPC requests.
-### Key: num_markers
+- **Default value**: `0`
+- **Constraints**: The value must be a positive double number.
+- **Description**: Specifies the timeout duration (in seconds) for RPC requests.
+
+### num_markers
+
 - **Required**: False
 - **Type**: int
 - **Default value**: None
-- **Constraints**: The minimum value is `1`. The maximum value is `256`
- -   **Description**: The number of markers is the number of coroutines to download the initial ledger
-### Key: dos_guard.whitelist.[]
+- **Constraints**: The minimum value is `1`. The maximum value is `256`.
+- **Description**: Specifies the number of coroutines used to download the initial ledger.
+
+### dos_guard.whitelist.[]
+
 - **Required**: False
 - **Type**: string
 - **Default value**: None
 - **Constraints**: None
- -   **Description**: List of IP addresses to whitelist for DOS protection.
-### Key: dos_guard.max_fetches
+- **Description**: The list of IP addresses to whitelist for DOS protection.
+
+### dos_guard.max_fetches
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 1000000
-- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`
- -   **Description**: Maximum number of fetch operations allowed by DOS guard.
-### Key: dos_guard.max_connections
+- **Default value**: `1000000`
+- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`.
+- **Description**: The maximum number of fetch operations allowed by DOS guard.
+
+### dos_guard.max_connections
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 20
-- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`
- -   **Description**: Maximum number of concurrent connections allowed by DOS guard.
-### Key: dos_guard.max_requests
+- **Default value**: `20`
+- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`.
+- **Description**: The maximum number of concurrent connections allowed by DOS guard.
```

Comment:
Concurrent connections for a specific IP address. Probably it's logical but could be better to specify to leave no place for any confusion.